int/config/check.go

```diff
@@ -100,21 +112,22 @@ func checkNSS(certPath string) error {
 		return nssCheckNonBlockingError("failed to instantiate the certutil runner", err)
 	}
 
-	manager := nss.NewManager([]string{}, service, logger.Logger)
+	loggerInstance := &NSSManagerLogger{}
+	manager := nss.NewManager([]string{}, service, loggerInstance)
```

Comment:
Why can't you pass logger here instead of loggerInstance ?

Reply:
because `logger` is a package now. the `nss.NewManager` require a interface that implement `Debugf` and `Errorf`. So I created a struct `NSSManagerLogger` that wrap the logger package.

I'd not be surprise if we can do better.

Reply:
https://github.com/massalabs/station/issues/1042